### ISSUE TITLE
Guidelines FOR more people to use OP Network

### ISSUE THREAD

**Issue by ugggfhbcnb** [ethereum-optimism/optimism#1844]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by maurelian** [comment]
Hi, thanks for opening this issue. 
> update the guidelines on the official website

Can you be more precise about what content should be updated?

**Comment by ugggfhbcnb** [comment]
Hi Maurelian,

Thank you for your reply, my suggestion is to update the USER DOCS of the website to guide more people join the OP Network because many of them use TrustWallet and TokenPocket. and also using short video or GIF of guidelines for easy understanding.

Thanks again.

**Comment by t0od** [comment]
> Hi Maurelian,
> 
> Thank you for your reply, my suggestion is to update the USER DOCS of the website to guide more people join the OP Network because many of them use TrustWallet and TokenPocket. and also using short video or GIF of guidelines for easy understanding.
> 
> Thanks again.

Yes, I agree with this. In China, a large number of users use mobile wallets, and they rarely use computers

**Comment by smartcontracts** [comment]
Ah okay. I do think that it would be a good idea to have more guides for using Optimistic Ethereum on different wallets.
